dist/configs/config/server.cfg

```diff
@@ -4,9 +4,18 @@
 // 0 - advertise everything, the server will be added to the public server list and information about it will be displayed to players
 // 1 - don't advertise but reply to status queries, this allows players to join and see what map is being played, how many people are playing, etc.
 // 2 - don't reply to status queries but accept connections, players can connect but not see server information
-// 3 - only accept LAN connections
 set server.private 0
 
+// allow source networks for incoming packets
+// 0 - loopback only
+// 1 - LAN
+// 2 - Internet
+set sv_networkScope 2
+
+// contact information to reach out to the server admin when needed: forum or chat nickname, mail address…
+// it is requested to not be empty for public servers
+set contact ""
```

Comment:
A commented-out example seems better. That way the config wouldn't overwrite a value set elsewhere. Also it is the convention used in the rest of the file, e.g. for the password cvar

Reply:
OK for a comment, but not OK for an example, or people will just uncomment and will silence the warning with the example (as bad as not doing anything).